user_tools/src/spark_rapids_pytools/wrappers/databricks_azure_wrapper.py

```diff
@@ -123,6 +124,84 @@ def qualification(cpu_cluster: str,
                              wrapper_options=wrapper_qual_options,
                              rapids_options=rapids_options).launch()
 
+    @staticmethod
+    def profiling(gpu_cluster: str = None,
+                  worker_info: str = None,
+                  eventlogs: str = None,
+                  profile: str = None,
+                  local_folder: str = None,
+                  remote_folder: str = None,
+                  tools_jar: str = None,
+                  credentials_file: str = None,
+                  jvm_heap_size: int = 24,
+                  verbose: bool = False,
+                  **rapids_options) -> None:
+        """
+        The Profiling tool analyzes both CPU or GPU generated event logs and generates information
+        which can be used for debugging and profiling Apache Spark applications.
+        :param  gpu_cluster: The Databricks-cluster on which the Spark applications were executed. The argument
+                can be a Databricks-cluster or a valid path to the cluster's properties file (json format)
+                generated by the databricks-CLI. If missing, then the argument worker_info has to be provided.
+        :param  worker_info: A path pointing to a yaml file containing the system information of a
+                worker node. It is assumed that all workers are homogenous.
+                If missing, the wrapper pulls the worker info from the "gpu_cluster".
+        :param  eventlogs: Event log filenames or ABFS (Azure Blob File System) storage directories
+                containing event logs (comma separated). If missing, the wrapper reads the Spark's
+                property `spark.eventLog.dir` defined in `gpu_cluster`. This property should be included
+                in the output of `databricks clusters get [--cluster-id CLUSTER_ID| --cluster-name CLUSTER_NAME]`.
+                Note that the wrapper will raise an exception if the property is not set.
+        :param profile: A named Databricks profile to get the settings/credentials of the Databricks CLI.
+        :param aws_profile: A named AWS profile to get the settings/credentials of the AWS account.
```

Comment:
It should be azure instead of aws. Is this required though?

Reply:
It is not required. Thank you! I will remove this.